packages/edit-site/src/components/global-styles/style.scss

```diff
@@ -53,15 +53,29 @@
 }
 
 .edit-site-global-styles-screen-colors {
-	margin: $grid-unit-20;
+	margin: $grid-unit-10 $grid-unit-20 $grid-unit-20;
```

Comment:
This no longer matches `.edit-site-global-styles-screen-typography`. We should probably combine into a new shared class.

Reply:
I updated in https://github.com/WordPress/gutenberg/pull/60031/commits/775cca4aa70b588260fb6ed9caa5324f8f3168dc; there are a few nuances between them that I didn't want to dive too deep into. Perhaps worth a follow-up refactor, now that these two panels are coming closer together.